packages/dashboard-message-bus/lib/message/types.ts

```diff
@@ -0,0 +1,44 @@
+export interface Message {
+  id: number;
+  type: string;
+  payload: any;
+}
+
+export interface DashboardProviderMessage extends Message {
```

Comment:
Each of these classes should probably have a docstring indicating what kind of message they're for